### PR TITLE
add senza misura to data.METERSIGN

### LINKED ISSUE #731: Add senza misura X to data.METERSIGN
We have no easy way to encode the "X" sometimes used to indicate a _senza misura_ with `meterSig`. 
We should add a value "senza" (or something similar) to `data.METERSIGN`.

Example (from Gould):
senza

In SMuFl:
https://smufl-browser.edirom.de/index.html?glyphname=timeSigX

### PR DESCRIPTION
**Pull request by rettinghaus**
fixes #731

### CODE REVIEW

File: source/modules/MEI.xml [hunk 1]
```diff
@@ -2019,6 +2019,9 @@
         <valItem ident="cut">
           <desc>Cut time; i.e. 2/2.</desc>
         </valItem>
+        <valItem ident="senza">
+          <desc>Senza misura</desc>
```
Comment (ahankinson) [inline source/modules/MEI.xml]: How does this differ from the `invis` value in `data.METERFORM`?

Comment (rettinghaus) [reply source/modules/MEI.xml]: Please see #731. It is meant to specifically encode [this symbol](https://smufl-browser.edirom.de/index.html?glyphname=timeSigX):
 


See Gould pp. 611–615

Comment (ahankinson) [reply source/modules/MEI.xml]: Ah, Ok. I was going by this:

https://usermanuals.musicxml.com/MusicXML/Content/EL-MusicXML-senza-misura.htm

Given that the other options in this are the "cut" and "common", perhaps something like "open" (from the SMuFL description "Open time signature") or just "x"? (not that I'm necessarily a fan of single-letter attribute values, but it is the most descriptive)  

Given that the MusicXML interpretation is an invisible time signature, but you definitely intend a visible one, maybe making that more explicit is a good thing. We can still keep "senza misura" in the description to make it clear...

Comment (ahankinson) [reply source/modules/MEI.xml]: Also this example, where the time signature has been hidden:

https://repertoire-explorer.musikmph.de/en/product/holst-gustav-17/

Comment (rettinghaus) [reply source/modules/MEI.xml]: I agree. `open` is a better value here.

Comment (ahankinson) [reply source/modules/MEI.xml]: Only in music notation does "X" mean "open", but it is what it is. :) 

Thanks!

File: source/modules/MEI.xml [hunk 2]
```diff
@@ -2019,6 +2019,9 @@
         <valItem ident="cut">
           <desc>Cut time; i.e. 2/2.</desc>
         </valItem>
+        <valItem ident="open">
+          <desc>Senza misura</desc>
```
Comment (ahankinson) [inline source/modules/MEI.xml]:
```suggestion
          <desc>Open time signature, i.e., Senza misura. See Gould pp. 611–615.</desc>
```

**Review by ahankinson — APPROVED** [review]

**Pull request merged**